beaconChain/tests/helpers/stateTransitionHelpers.test.ts

```diff
@@ -101,44 +102,38 @@ describe("Clamp", () => {
 
 describe("isPowerOfTwo", () => {
   it("0 should return false", () => {
-    const result = isPowerOfTwo(0);
+    const result = isPowerOfTwo(new BN(0));
     assert.equal(result, false, "Should have returned false!");
   });
 
   it("1 should return true", () => {
-    const result = isPowerOfTwo(1);
+    const result = isPowerOfTwo(new BN(1));
     assert.equal(result, true, "Should have returned true!");
   });
 
   it("2 should return true", () => {
-    const result = isPowerOfTwo(2);
+    const result = isPowerOfTwo(new BN(2));
     assert.equal(result, true, "Should have returned true!");
   });
 
   it("3 should return false", () => {
-    const result = isPowerOfTwo(3);
+    const result = isPowerOfTwo(new BN(3));
     assert.equal(result, false, "Should have returned false!");
   });
 
   it("Numbers close to 2**32 should return false", () => {
     for (let i: int = 2; i < 53; i++) {
```

Comment:
This for loop should go from 2 to 257 since we want to make sure this works for all 256 bit numbers.